content/community/getting-involved/developing/system.html

```diff
@@ -302,8 +302,8 @@ <h4>
 Several preference applications (aka preflets) could be redesigned. Furthermore, there might still be code that does not yet use our layout API. This work may include (but is not limited to):
 <ol>
 <li>combining Keymap and Keyboard</li>
-<li><a href="https://dev.haiku-os.org/ticket/6983">#6983</a>Printer</li>
-<li><a href="https://dev.haiku-os.org/ticket/6206">#6206</a>integrate scrollbar options into a new Appearance preflet</li>
+<li><a href="https://dev.haiku-os.org/ticket/6983">#6983</a> Printer</li>
+<li><a href="https://dev.haiku-os.org/ticket/6206">#6206</a> Integrate scrollbar options into a new Appearance preflet</li>
```

Comment:
This one already has a pending patch on Gerrit and several devs have said they don't want options for it. I suggest not recommending to beginners to work on sonething most developers don't want to merge, and that someone else already has done.